pythia6.sh

```diff
@@ -33,6 +33,7 @@ module load BASE/1.0
 # Our environment
 setenv PYTHIA6_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
 prepend-path LD_LIBRARY_PATH \$::env(PYTHIA6_ROOT)/lib
+prepend-path AGILE_GEN_PATH \$::env(PYTHIA6_ROOT)
 $([[ ${ARCHITECTURE:0:3} == osx ]] && echo "prepend-path DYLD_LIBRARY_PATH \$::env(PYTHIA6_ROOT)/lib")
```

Comment:
It adds it to the environment variable which is used by AGILe to search for generators (AGILe is an interface to run (old) generators and produce HepMC output). The corresponding PR is in preparation (https://github.com/qgp/alidist/tree/add_agile) but there's no point adding this before we have PYTHIA.